near/omni-bridge/src/lib.rs

```diff
@@ -704,6 +727,11 @@ impl Contract {
                 ),
             )
         } else {
+            if let OmniAddress::Btc(_) = fast_fin_transfer_msg.recipient {
+                let btc_account_id = self.get_native_token_id(ChainKind::Btc);
+                assert_eq!(token_id, btc_account_id);
```

Comment:
Why not `require!` with a message?

Reply:
fixed: https://github.com/Near-One/omni-bridge/pull/348/commits/b759a597e7c21ab84beab8c9df15a2c16cc1433f